pyoozie/tags.py

```diff
@@ -613,6 +615,87 @@ def _xml(self, doc, tag, text, on_next, on_error):
         return doc
 
 
+class Serial(_WorkflowEntity):
+    """Sequence of actions to execute (implemented by chaining actions and 'OK' transitions)"""
+
+    def __init__(self, *actions, **kwargs):
+        # type: (*_WorkflowEntity, **_WorkflowEntity) -> None
+        super(Serial, self).__init__(xml_tag=None, on_error=kwargs.get(str('on_error')))
+        self.__actions = tuple(copy.deepcopy(actions))  # type: typing.Tuple[_WorkflowEntity, ...]
```

Comment:
This is something we're going to have to document carefully. Adding an action to a `Serial` / `Parallel` makes it very hard to adjust the action afterwards, so certain "obvious" patterns will be wrong.
Example:
 ❌   create 3 jobs, run in parallel, create 3 notifications & attach to jobs
 ✅   create 3 jobs, create 3 notifications & attach to jobs, run in parallel